pytest_fixtures/authsource_fixtures.py

```diff
@@ -0,0 +1,207 @@
+"""Test class for Active Directory Feature
```

Comment:
I'd say its sufficient to call this module something like `pytest_fixtures/satellite_auth.py`, as the path contains 'fixtures' so we don't need to repeat it in the module name.

Being a bit more explicit in the naming of the module, it should be obvious these are fixtures related to authentication in Satellite - we could additionally later have `pytest_fixtures/virtwho_auth.py` for fixtures related to virt-who hypervisor authentication (as an example).